suites/002-file-system/delete.robot

```diff
@@ -38,6 +40,24 @@ Create And Delete A File
 
     [Teardown]    Teardown Test Path
 
+Create And Delete A Directory
+    [Documentation]    Create And Delete A Directory
+    ...
+    ...    Client1 creates a directory and deletes it.
+
+    [Setup]    Setup Test Path
+
+    client1.Create Directory    path=${DIR_PATH}
+    client1.Wait Until Created    path=${DIR_PATH}
```

Comment:
As a style thing, maybe the some vertical alignment might be nice (in future changes). For example the "path=..." could be aligned, which could make this nicer to read. would that trip up the robot linter?

Reply:
G mentions robocop can autoformat into tables. Lets try that later.